pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -430,7 +433,18 @@ public static String createNewOncHistoryDetail(@NonNull String medicalRecordId,
             throw new RuntimeException("Error adding new oncHistoryDetail for medicalRecord w/ id " + medicalRecordId,
                     results.resultException);
         } else {
-            return (String) results.resultValue;
+            String oncHistoryDetailId = (String) results.resultValue;
```

Comment:
adding upsert here to add new oncHistoryDetail object to ES even if it is empty and only has oncHistoryDetailId
@gmakharat @Anzori-Ghurtchumelia